### PR TITLE
Documented running custom state at highstate in Salt Guide

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/uyuni-project/uyuni-docs/pull/807 to be merged first.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes https://github.com/uyuni-project/uyuni/issues/2704

### CODE REVIEW

**Comment by keichwa** [comment]
Is this to be backported to 4.1?

**Comment by Loquacity** [comment]
> Is this to be backported to 4.1?

No, let's keep it for 4.2. Is this going to cause conflicts with my PR #807 though?

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

>> Is this to be backported to 4.1?
>
> No, let's keep it for 4.2.

Ok, fine.

> Is this going to cause conflicts with my PR #807 though?

I guess your PR is a rather complex custom Salt state, and to execute it
at highstate (if wanted), you'd have to reference it similar to the
example shown in my PR.  I hope our developers will take a closer look,
though ;-)

Whether there will be plain git conflicts, we'll see.  I can wait with
merging until your PR is in!

-- 
Karl Eichwalder

**Comment by Loquacity** [comment]
> Is this going to cause conflicts with my PR #807 though?
> I guess your PR is a rather complex custom Salt state, and to execute it at highstate (if wanted), you'd have to reference it similar to the example shown in my PR. I hope our developers will take a closer look, though ;-) Whether there will be plain git conflicts, we'll see. I can wait with merging until your PR is in!

Well, yes, but it also creates a new page for Custom Salt States, which is where I think this content might need to be. It might work better if you do your PR against mine.

**Review by jcayouette — APPROVED** [review]

**Comment by Loquacity** [comment]
PR#807 is merged, so this should be good to go now.

File: modules/salt/pages/salt-states.adoc [hunk 1]
```diff
@@ -70,6 +70,17 @@ Apply the highstate to all clients in the group:
...
+This short example maps the [literal]``test`` state to the system group [literal]``12``:
+
+----
+# /srv/salt/top.sls
+base:
+  'group_ids:12':
+    - match: pillar
+    - test
+----
+
```
Comment (Loquacity) [inline modules/salt/pages/salt-states.adoc:83]: So this probably needs to go in the new Custom Salt States section now.

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: modules/salt/pages/salt-states.adoc (hunk 1 above)
Comment (keichwa) [reply modules/salt/pages/salt-states.adoc]: I moved it. Please, recheck!

**Review by agraul — APPROVED** [review]

**Review by Loquacity — APPROVED** [review]

**Pull request merged**